docs/INSTALL.TXT

```diff
@@ -5,19 +5,20 @@ Welcome to Zen Cart(R) E-Commerce
 THE FOLLOWING IS A VERY SIMPLIFIED INSTALLATION DOCUMENT.
 You can find online documentation at: https://docs.zen-cart.com
 
-WE SUGGEST YOU USE THE PDF "Implementation Guide" PDF FILE IN THE /docs FOLDER FOR FULLY DETAILED INSTRUCTIONS.
+WE SUGGEST YOU USE ONLINE DOCS AND THE PDF "Implementation Guide" 
+
+https://docs.zen-cart.com/release/implementation-guide-v157.pdf
+
+FOR FULLY DETAILED INSTRUCTIONS.
 
 
 Zen Cart has a built-in automatic installation system. However to use this you must first unpack and upload the code to a compatible web server:
 
 1. Extract the Zen Cart ZIP file (and tell the unzip utility to *retain* folder structures)
 2. Upload the Zen Cart files and folders to your server using your FTP program
-3. You must make some folders writable. The details are in the /docs/Implementation-Guide.pdf and the HTML-formatted documentation in the /docs/ folder
+3. You must make some folders writable. The details are in the Implementation-Guide.pdf and the online documentation. 
```

Comment:
Here you mention Implementation-Guide.pdf here but above called it implementation-guide-v157.pdf
should you include a link? Or point them to Docs>release Documentation>Implementation Guide (PDF)

Reply:
Fixed thanks.